.changeset/three-rats-shake.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-test-data/channel': minor
+'@commercetools-test-data/zone': minor
```

Comment:
```suggestion
'@commercetools-test-data/zone': major
```

Reply:
I think this will bump all to major, as we are using `fixed` mode in Changeset.

As it is now it should be fine. The package will be published as version `4.x`.